ops/setup.py

```diff
@@ -22,7 +22,7 @@
     keywords=["juju", "charming", "kubernetes", "ops", "framework", "interface"],
     name="ops.interface_kube_control",
     packages=find_namespace_packages(include=["ops.*"]),
-    url="https://github.com/juju-solutions/interface-kube-control/blob/HEAD/ops",
+    url="https://github.com/charmed-kubernetes/interface-kube-control/blob/HEAD/ops",
```

Comment:
Uh... no let's leave it pointing at the original for the time being

If we need to move the archived repo to the `canonical` org so be it...